packit_service/worker/handlers/abstract.py

```diff
@@ -114,9 +114,9 @@ def _clean_workplace(self):
 
     def pre_check(self) -> bool:
         """
-        Validate the input.
-
-        :return: False if we can skip the run.
+        Implement this method for those handlers, where you want to check if the properties are
+        correct. If this method returns False during runtime, execution of service code is skipped.
+        :return:
```

Comment:
Is this `:return:` still needed?

Reply:
Copied from the other one... I'll fix both of them.